vignettes/bcva.Rmd.upstream

```diff
@@ -0,0 +1,391 @@
+---
+title: BCVA data comparison between Bayesian and frequentist MMRMs
+date: "`r format(Sys.time(), '%d/%m/%Y')`"
+output:
+  rmarkdown::html_document:
+    theme: spacelab
+    highlight: default
+    toc: yes
+    toc_float: yes
+    number_sections: true 
+vignette: >
+  %\VignetteIndexEntry{BCVA data comparison between Bayesian and frequentist MMRMs}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options:
+  chunk_output_type: console
+  markdown:
+    wrap: 72
+---
+
+```{r setup, eval=TRUE, include=FALSE, warning=FALSE, message=FALSE}
+library(knitr)
+knitr::opts_chunk$set(
+  echo = TRUE,
+  collapse = TRUE,
+  warning = FALSE,
+  message = FALSE,
+  prompt = TRUE,
+  comment = "#>",
+  size = "scriptsize",
+  fig.height = 5,
+  fig.width = 6,
+  dpi = 320,
+  fig.align = "center",
+  fig.path = "bcva_figures/"
+)
+```
+
+# About {.unnumbered}
+
+This vignette uses the `bcva_data` dataset from the `mmrm` package to compare a Bayesian MMRM fit, obtained by `brms.mmrm::brm_model()`, and a frequentist MMRM fit, 
```

Comment:
Very minor: spelling in this vignette, with and without hyphen, "`mmrm` package" vs. "`mmrm`-package"

Reply:
Removed the hyphen in 56fbdc501ec8d7956b7f809d7faf0c9c4ebb91a5.